### ISSUE TITLE
where are drush and drupal (console) setup?

### ISSUE THREAD

**Issue by laurencefass** [iBobik/drupal-composer-docker#3]
hi i've installed a new project via composer

drush --version and drupal --verison are not recognised. where are they set up? drush seems to be installed locally which is vendor/drush/drush/... so i can set up a link to it but thats not what i was expecting.

which drush returns nothing
which drupal returns nothing.

thanks

**Comment by tanmayk** [comment]
Yeah, drush is not getting installed. Below is the output of drush command I tried:

```
#!/bin/bash -eo pipefail
drush config-get system.site name
/bin/bash: drush: command not found

Exited with code exit status 127
CircleCI received exit code 127
```
